src/pages/guides/uxp_guide/uxp-misc/manifest-v5/index.md

```diff
@@ -0,0 +1,282 @@
+---
+keywords:
+  - Creative Cloud
+  - API Documentation
+  - UXP
+  - Plugins
+  - JavaScript
+  - JSON
+  - Manifest
+  - Scripting
+title: Manifest v5
+description:
+---
+
+# UXP Manifest v5
+
+Manifest v5 gives developers access to a new plugin permissions model and WebViews in modal dialogs.
+
+Upgrade your plugin to use the latest manifest feature set by changing the `manifestVersion` element. 
+```json
+"manifestVersion": 5
+```
+## Example manifest
+```json
+{
+    "manifestVersion": 5,
+    "id": "YOUR_ID_HERE",
+    "name": "Name of your plugin",
+    "version": "1.0.0",
+    "main": "index.html",
+    "host": {
+        "app": "PS",
+        "minVersion": "23.0.0"
+    },
+    "entrypoints": [
+        {
+            "type": "command",
+            "id": "commandFn",
+            "label": {
+                "default": "Show A Dialog"
+            }
+        },
+        {
+            "type": "panel",
+            "id": "panelName",
+            "label": {
+                "default": "Panel Name"
+            },
+            "minimumSize": {"width": 230, "height": 200},
+            "maximumSize": {"width": 2000, "height": 2000},
+            "preferredDockedSize": {"width": 230, "height": 300},
+            "preferredFloatingSize": {"width": 230, "height": 300},
+            "icons": [
+                {"width":23,"height":23,"path":"icons/dark.png","scale":[1,2],"theme":["darkest","dark","medium"]},
+                {"width":23,"height":23,"path":"icons/light.png","scale":[1,2],"theme":["lightest","light"]}
+            ]
+        }
+    ],
+    "icons": [
+        { "width": 23, "height": 23, "path": "icons/icon_D.png", "scale": [ 1, 2 ], "theme": [ "dark", "darkest" ], "species": [ "generic" ] },
+        { "width": 23, "height": 23, "path": "icons/icon_N.png", "scale": [ 1, 2 ], "theme": [ "lightest", "light" ], "species": [ "generic" ] }
+    ],
+    "featureFlags": {
+        "enableExperimentalWebviewSupport" : true
```

Comment:
Double check this, but IIRC, this flag is no longer required for webviews to be enabled -- they should be enabled by default.

Reply:
yep you're right!